desc/geometry/curve.py

```diff
@@ -807,6 +808,76 @@ def compute_length(self, X_n=None, Y_n=None, Z_n=None, grid=None):
         theta = transform.grid.nodes[:, 2]
         return jnp.trapz(T, theta)
 
+    @classmethod
+    def from_XYZCurve(cls, xyz_curve, N=10, grid=None):
```

Comment:
probably don't need both from and to methods. For profiles we have `to_*` methods in the base class, and then each subclass just has a `from_values` method to fit from raw points, we could probably due something similar here

Reply:
because we have the base classes in a separate file, I dont think we can do a similar thing without running into circular imports. I at least could not figure it out

Reply:
we could just do a local import of the necessary class in the `to_*` method in the base class?